examples/05-cursor-selections/selection-blocks/README.md

```diff
@@ -1 +1,3 @@
-# Highlighting Blocks in Selection
\ No newline at end of file
+# Highlighting Blocks in Selection
+
+TODO: same as text-cursor block. Perhaps replace both by 1 single example
```

Comment:
Good idea, can just use a different background color for block with text cursor vs blocks in selection.

Reply:
Ah, not what I meant, see comments on the other related example